clap_mangen/src/render.rs

```diff
@@ -107,10 +107,24 @@ pub(crate) fn options(roff: &mut Roff, cmd: &clap::Command) {
         }
 
         let mut body = vec![];
+        let mut help_written = false;
         if let Some(help) = opt.get_long_help().or_else(|| opt.get_help()) {
+            help_written = true;
             body.push(roman(help));
         }
 
+        let possibles = &opt.get_possible_values();
+
+        if !(possibles.is_empty() || opt.is_hide_possible_values_set()) {
+            if help_written {
+                // It looks nice to have a separation between the help and the values
+                body.push(Inline::LineBreak);
+            }
+
+            let possible_vals = possibles.iter().filter(|pos| !pos.is_hide_set()).collect();
```

Comment:
Should we do the filtering before we check for empty?

Reply:
Good catch